website/server/controllers/api-v3/chat.js

```diff
@@ -177,6 +177,11 @@ api.postChat = {
     }
 
     const newChatMessage = group.sendChat(req.body.message, user);
+
+    if (user.flags && user.flags.verifiedUsername === true) {
```

Comment:
is this necessary? Because the same block of code in `messageDeafult` should be used when sending chat as well so i don't think this is necesssary